spec.bs

```diff
@@ -464,6 +459,12 @@ An {{AppHistoryNavigateEvent}} also has an associated {{Promise}}-or-null <dfn f
   1. Let |result| be the result of [=dispatching=] |event| at |appHistory|.
   1. If [=this=]'s [=relevant global object=]'s [=Window/browsing context=] is null, then return false.
      <p class="note">This can occurr if an event listener disconnected the <{iframe}> corresponding to [=this=]'s [=relevant global object=].</p>
+  1. If |event|'s [=AppHistoryNavigateEvent/navigation action promise=] is non-null, then:
```

Comment:
`navigation action promise` will only be non-null here if `respondeWith()` was called, right?

Reply:
Exactly, it's basically synonymous with that.